src/main/java/org/opentripplanner/updater/DataSource.java

```diff
@@ -0,0 +1,26 @@
+package org.opentripplanner.updater;
+
+import java.util.List;
+
+/**
+ * TODO clarify thread safety.
+ * It appears that update() and getStations() are never called simultaneously by different threads, but is not stated.
```

Comment:
Should we trust these are never called from different threads?

Reply:
I think it is enough to state that you assume that the updaters are NOT run in parallell.